e2e-tests/pitr/run

```diff
@@ -18,11 +18,13 @@ run_recovery_check_pitr() {
     local compare=$4
     local time_now=$5
     local dest=$6
+    local gtid=$7
```

Comment:
can you please run `shfmt -bn -ci -s -w e2e-tests/pitr/run` and commit file again?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/801/commits/7ff994d26779ab25ce61ab7f00c9439d5fa70bab